crates/chia-tools/src/bin/benchmark-condition-costs.rs

```diff
@@ -67,14 +67,20 @@ fn cons_condition(allocator: &mut Allocator, current_ptr: NodePtr) -> Result<Nod
 fn create_conditions(
     allocator: &mut Allocator,
     condition: &ConditionTest,
+    reps: u32,
 ) -> Result<NodePtr, EvalErr> {
     let mut rest = allocator.nil();
     for arg in condition.args.iter().rev() {
         rest = allocator.new_pair(*arg, rest)?;
     }
     let opcode = allocator.new_small_number(condition.opcode as u32)?;
     let cond_list = allocator.new_pair(opcode, rest)?;
-    allocator.new_pair(cond_list, allocator.nil())
+
+    let mut ret = NodePtr::NIL;
+    for _ in 0..reps {
+        ret = allocator.new_pair(cond_list, ret)?;
+    }
+    Ok(ret)
 }
```

Comment:
I think you want to remove this change now

Reply:
done